augur/merge.py

```diff
@@ -168,16 +221,102 @@ def run(args: argparse.Namespace):
     metadata: Optional[List[NamedMetadata]] = None
     output_columns: Optional[Columns] = None
     output_source_column: Optional[Callable[[str], str]] = None
+    sequences: Optional[List[Union[NamedSequenceFile, UnnamedSequenceFile]]] = None
+    named_sequences: Optional[List[NamedSequenceFile]] = None
 
     if args.metadata:
         metadata = get_metadata(args.metadata, args.metadata_id_columns, args.metadata_delimiters)
+
+    if args.sequences:
+        sequences = list(get_sequences(args.sequences))
```

Comment:
_minor_

This `list()` is necessary only incidentally because it's more convenient to implement `get_sequences()` as a generator. Consider making `get_sequences()` return a list so its caller (i.e. right here) doesn't have to care about its internals, even if it uses a generator internally.